.github/workflows/push-docker.yml

```diff
@@ -140,3 +140,7 @@ jobs:
         tags: ${{ steps.meta3.outputs.tags }}
         labels: ${{ steps.meta3.outputs.labels }}
         platforms: linux/amd64,linux/arm64/v8
+    - name: Build and Push Helm Chart
+      run: |
+        helm package chart/stirling-pdf
+        helm push stirling-pdf-chart-1.0.0.tgz oci://registry-1.docker.io/frooodle
```

Comment:
Any issue here having the version tag always be 1.0.0?

Reply:
Yeah this is not ideal, in an ideal world you would version the chart and container image independently, but since I don't expect the chart to be changed that I think it won't be a problem.